src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3524,6 +3530,9 @@ private BoundNode BindConstructorBody(ConstructorDeclarationSyntax constructor,
                                                       null :
                                                       bodyBinder.BindExpressionBodyAsBlock(constructor.ExpressionBody,
                                                                                            constructor.Body == null ? diagnostics : BindingDiagnosticBag.Discarded));
+
+            bool hasAnyRecordConstructors() =>
```

Comment:
>hasAnyRecordConstructors [](http://example.com/codeflow?start=17&length=24)

I think the name is confusing. The body is checking for a primary constructor, not just any constructor.